main.py

```diff
@@ -103,6 +103,15 @@ class Release:
     release_type: str
 
 
+@dataclass
+class ReleaseTag:
+    version: str
+    page: str
+    release_type: str
+    tag_url: str = None
```

Comment:
If you want `None` as default value (not sure if that's necessary), you need `Optional[str]`